LSE-61.tex

```diff
@@ -1174,6 +1178,93 @@ \subsubsection{All-Sky Visualization of Data Releases}
 OSS-REQ-0136:
 Co-added Exposures \newline
 
+\paragraph{Produce All-Sky HiPS Map}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0379}
+\textbf{ID:} DMS-REQ-0379
```

Comment:
This should be Priority 1b.  To be a bit finer-grained, this SHOULD be testable with mini-survey data, which is consistent with 1b, but MUST be available in time for DR1.  I'm uncomfortable with the "three year delay" in Priority 2.

Reply:
OSS-REQ-0136 (Level 2 coadds) and OSS-REQ-0391 (follow DPDD) appear to be the best targets for `<deriveReqt>`.  The latter requires the inclusion of HiPS in the DPDD, which we should do.